front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -244,7 +247,7 @@ const PacedTrainItem = ({
           </div>
         )}
         <TimetableItemActions
-          selectPathProjection={selectPathProjection}
+          selectPathProjection={() => selectPathProjection()}
```

Comment:
nit: it works and is more concise
```suggestion
          selectPathProjection={selectPathProjection}
```

Reply:
sure thanks !